js/npm/sequelize/DataTypes.hx

```diff
@@ -28,14 +28,14 @@ class DataTypes {
 	}
 	/**
 	 * A 32 bit integer. 
-	 * Available properties: `length`, `unsigned:Bool`, `zerofill:Bool`
+	 * Available properties: `length:Int`, `unsigned:Bool`, `zerofill:Bool`
 	 */
 	static public inline function INTEGER(?options:{}):DataTypes {

```

Comment:
In fact, in JS, `Sequelize['BIGINT'](options)` == `Sequelize.BIGINT(options)`
Then if the function can have the same name in Haxe and in JS (`BIGINT`), you don't need to `static public inline`, you can just do :
```haxe
static public function BIGINT(?options:{}): DataTypes;
```
You usually need `static inline` when the identifier is incompatible with Haxe and _really_ requires the use of `[]` (for example: `stuff["foo-bar"]` or `express["static"]` because `static` is a Haxe keyword)